src/main/java/com/bunq/sdk/model/generated/object/DraftPaymentEntry.java

```diff
@@ -0,0 +1,177 @@
+package com.bunq.sdk.model.generated.object;
+
+import com.bunq.sdk.model.BunqModel;
+import com.bunq.sdk.model.MonetaryAccountReference;
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import java.util.List;
+
+/**
+ */
+public class DraftPaymentEntry extends BunqModel {
+
+  /**
+   * The amount of the payment.
+   */
+  @Expose
+  @SerializedName("amount")
+  private Amount amount;
+
+  /**
+   * The LabelMonetaryAccount containing the public information of the other (counterparty) side
+   * of the DraftPayment.
+   */
+  @Expose
+  @SerializedName("counterparty_alias")
+  private MonetaryAccountReference counterpartyAlias;
+
+  /**
+   * The description for the DraftPayment. Maximum 140 characters for DraftPayments to external
+   * IBANs, 9000 characters for DraftPayments to only other bunq MonetaryAccounts.
+   */
+  @Expose
+  @SerializedName("description")
+  private String description;
+
+  /**
+   * Optional data to be included with the Payment specific to the merchant.
+   */
+  @Expose
+  @SerializedName("merchant_reference")
+  private String merchantReference;
+
+  /**
+   * The Attachments attached to the DraftPayment.
+   */
+  @Expose
+  @SerializedName("attachment")
+  private List<AttachmentMonetaryAccountPayment> attachment;
+
+  /**
+   * Whether or not sending a bunq.to payment is allowed.
+   */
+  @Expose
+  @SerializedName("allow_bunqto")
+  private Boolean allowBunqto;
+
+  /**
+   * The id of the draft payment entry.
```

Comment:
`id`, please convert to uppercase or change the uppercase ones to lower case

Reply:
generated